projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -78,6 +80,19 @@ public function register_routes() {
 			)
 		);
 
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/last-status',
+			array(
+				array(
+					'methods'             => WP_REST_Server::EDITABLE,
+					'callback'            => array( $this, 'update_last_status' ),
+					'permission_callback' => array( $this, 'update_last_status_permissions_check' ),
+					'args'                => $this->get_last_status_params(),
```

Comment:
It's fine to have those in a function, but don't feel the need to do it because the other ones are. I just did that because they were used in two endpoints.

Reply:
I created a new function because the same fields are in the POST arguments as well as in the item schema and so maybe I could have merged the arrays (that I didn't).